### PR TITLE
inspire-utils bump and check packages

### PR DESCRIPTION
**Pull request by DonHaul**

### CODE REVIEW

**Comment by DonHaul** [comment]
`urllib` is not even being used. removed it

File: README.md [hunk 1]
```diff
@@ -18,13 +19,30 @@
...
+
+Tests should be run both for python 2 and python 3.
+For python 2 its recommended to run them through docker:
+```bash
+# building the image python 3 and python 2
+docker build -f Dockerfile.py2   -t inspire-utils:py2 .
+docker run -it inspire-utils:py2 /bin/bash
+
+# inside the container
+pytest tests
```
Comment (drjova) [inline README.md:40]: docker run inspire-utils:py2 pytest tests

**Review by drjova — COMMENTED** [review]
just a small update

File: README.md (hunk 1 above)
Comment (DonHaul) [reply README.md]: updated

**Comment by DonHaul** [comment]
Found `python:2.7.18-buster` that already includes all the dependencies needed for `lxml` - updated docker image

**Review by drjova — APPROVED** [review]

**Pull request merged**